disable the check_chroot_filelist calls in unbound-checkconf since we copy
/etc/unbound to the chroot dir in the init script
Index: unbound-1.0.0/smallapp/unbound-checkconf.c
===================================================================
--- unbound-1.0.0.orig/smallapp/unbound-checkconf.c	2008-05-25 01:29:54.963462343 -0400
+++ unbound-1.0.0/smallapp/unbound-checkconf.c	2008-05-25 01:31:57.811471789 -0400
@@ -321,6 +321,7 @@
 		!is_dir(cfg->chrootdir)) {
 		fatal_exit("bad chroot directory");
 	}
+	/*
 	if(cfg->chrootdir && cfg->chrootdir[0]) {
 		char buf[10240];
 		buf[0] = 0;
@@ -334,6 +335,7 @@
 			fatal_exit("config file %s is not inside chroot %s",
 				buf, cfg->chrootdir);
 	}
+	*/
 	if(cfg->directory && cfg->directory[0] && !is_dir(
 		fname_after_chroot(cfg->directory, cfg, 0))) {
 		fatal_exit("bad chdir directory");
@@ -352,12 +354,14 @@
 		}
 	}
 
+	/*
 	check_chroot_filelist("file with root-hints", 
 		cfg->root_hints, cfg->chrootdir, cfg);
 	check_chroot_filelist("trust-anchor-file", 
 		cfg->trust_anchor_file_list, cfg->chrootdir, cfg);
 	check_chroot_filelist("trusted-keys-file", 
 		cfg->trusted_keys_file_list, cfg->chrootdir, cfg);
+	*/
 	/* remove chroot setting so that modules are not stripping pathnames*/
 	free(cfg->chrootdir);
 	cfg->chrootdir = NULL;
